PennMobile/DiningViewController2.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  DiningViewController2ViewController.swift
+//  PennMobile
+//
+//  Created by Victor Chien on 11/12/16.
+//  Copyright © 2016 PennLabs. All rights reserved.
+//
+
+import UIKit
+
+class DiningViewController2ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource{
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+
```

Comment:
Why do we need this class?  Both functions that are overrode have the same functionality as the super...  Delete this or add more functionality